pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -682,7 +682,7 @@ class ThinkingPart:
     signature: str | None = None
```

Comment:
I suppose the alternative to base64 encoding/decoding would be do `signature: str | bytes | None = None`, but I'm not sure what kind of ripple effects that would produce.

Reply:
Let's do that